roboflow/deployment.py

```diff
@@ -69,6 +70,14 @@ def add_deployment_parser(subparsers):
     deployment_delete_parser.add_argument("-a", "--api_key", help="api key")
     deployment_delete_parser.add_argument("deployment_name", help="deployment name")
 
+    deployment_log_parser.set_defaults(func=get_deployment_log)
+    deployment_log_parser.add_argument("-a", "--api_key", help="api key")
+    deployment_log_parser.add_argument("deployment_name", help="deployment name")
+    deployment_log_parser.add_argument(
+        "-d", "--duration", help="duration of log (from now) in seconds", type=int, default=3600
+    )
+    deployment_log_parser.add_argument("-f", "--follow", help="follow log output", action="store_true")
```

Comment:
Is it worth having a flag to show the last n lines of the log?

Reply:
Good idea! Added